app/scripts/templates/marketing_snippet.mustache

```diff
@@ -28,13 +29,30 @@
 
           <ul>
             <li>
-              <a class="marketing-link marketing-link-ios" href="https://app.adjust.com/2uo1qc?campaign=fxa-conf-page&adgroup=ios&creative=button&fallback=https://itunes.apple.com/app/apple-store/id989804926?pt=373246&ct=adjust_tracker&mt=8" data-marketing-id="spring-2015-android-ios-sync"><img width="152" height="45" src="{{ appStoreImage }}" alt="{{#t}}Download Firefox for iOS!{{/t}}" /></a>
+              <a class="marketing-link marketing-link-ios" href="{{ downloadLinkIos }}" data-marketing-id="{{ marketingId }}" data-marketing-type="ios"><img width="152" height="45" src="{{ appStoreImage }}" alt="{{#t}}Download Firefox for iOS!{{/t}}" /></a>
             </li>
             <li>
-              <a class="marketing-link marketing-link-android" href="https://app.adjust.com/2uo1qc?campaign=fxa-conf-page&adgroup=android&creative=button" data-marketing-id="spring-2015-android-ios-sync"><img width="152" height="45"  src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
+              <a class="marketing-link marketing-link-android" href="{{ downloadLinkAndroid }}" data-marketing-id="{{ marketingId }}" data-marketing-type="android"><img width="152" height="45"  src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
             </li>
           </ul>
         </div>
       {{/isOther}}
     </aside>
-  {{/showSignUpMarketing}}
+  {{/isSpring2015}}
+  {{#isAutumn2016}}
+    {{#isIos}}
+      <a class="marketing-link marketing-link-ios" href="{{ downloadLinkIos }}" data-marketing-id="{{ marketingId }}" data-marketing-type="ios"><img width="152" height="53" src="{{ appStoreImage }}" alt="{{#t}}Download Firefox for iOS!{{/t}}" /></a>
+    {{/isIos}}
+
+    {{#isAndroid}}
+      <a class="marketing-link marketing-link-android" href="{{ downloadLinkAndroid }}" data-marketing-id="{{ marketingId }}" data-marketing-type="android"><img width="152" height="45" src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
+    {{/isAndroid}}
+
+    {{#isOther}}
+      <div class="links cf">
```

Comment:
clearfix. Needed to prevent the "Why is this required" text from showing up smushed between the links to the stores since they both use float.